### PR TITLE
WiP: Run PHPCS on git commit

### PR DESCRIPTION
**Pull request by khalima**
# Run PHPCS on git commit

## What was done

* Added shell script to the theme pre-commit hook

## How to install

* Make sure your instance is up and running on latest dev branch.
  * `git pull origin dev`
  * `make fresh`
* Update the HDBT theme
  * `composer require drupal/hdbt:dev-UHF-X_phpcs_run_on_commit`
* Run `make drush-cr`

## How to test

* [ ] Check that this feature works
* [ ] Check that code follows our standards

## Designers review

* [ ] This PR does not need designers review
* [ ] This PR has been visually reviewed by a designer (Name of the designer)

## Other PRs

* Link to other PR

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-16.17.1
+16.18.0
```
Comment (Arkkimaagi) [inline .nvmrc:1]: This is outdated

File: .husky/pre-commit [hunk 2]
```diff
@@ -1,4 +1,19 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
+DIR=$(pwd)
+echo $(php --version)
+
+# Switch to MAKEFILE.
+
+if [[ "$DIR" == *"/public/themes/contrib/"* ]]; then
```
Comment (Arkkimaagi) [inline .husky/pre-commit:9]: Maybe we should check if the tools are available from Drupal before continuing to use them?

**Review by Arkkimaagi — CHANGES_REQUESTED** [review]
Left a couple of notes for when you get back to this.

**Comment by khalima** [comment]
Closing this as running phpcs manually on local env should be enough.

**Pull request closed without merging**